FBC/FunctionalEntities/RegistrationAuthorities.rdf

```diff
@@ -218,15 +225,15 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>registry</rdfs:label>
-		<skos:definition>a system, typically an information system, that records the registration of items</skos:definition>
+		<skos:definition>system, typically an information system, that records the registration of items</skos:definition>
```

Comment:
I'll work on this - ontologies typically do not define lists of anything, so 'record' is better, and registries may contain various kinds of information on a specific topic or about a specific thing, not "one kind of information' per se, which itself is somewhat misleading - for example, a registry about a business might include identifiers, ownership details, etc. which I would consider 'multiple kinds of information' but all about an individual business.

Reply:
Not a showstopper but I still prefer "one kind of information" to "relating to something" which your latest has.

Reply:
It could be a collection of information that is related to something, not necessarily exactly 'one kind' of homogeneous information.  Think about a church registry - there are records about various sacraments, prayer requests, people for whom a particular service is dedicated ... not one kind of information, but a collection of information about current and past parishoners or family members of parishoners.

Reply:
A church registry provides a formal record of certain activities of the church, in other words, but there is typically not a separate registry for baptisms vs. first communions vs. marriages vs. prayer requests, etc.  Yet, those records are hugely important for genealogy purposes.  They record events that are important to people's lives, and other things.  The other things are less important as time passes, typically, but the record outlives the people whose lives it documents things about.